packages/default-theme/src/pages/registration/confirm.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="sw-registration-confirm">
+    <SfHeading :level="2" :title="$t('Account confirmation')" />
+    <SfLoader class="sw-loader" :loading="loading">
+      <div class="sw-registration-confirm__form">
+        <SfHeading
+          v-if="errors.length && !success"
+          :level="3"
+          :title="$t('Something went wrong...')"
+        />
+        <SfHeading
+          v-if="success"
+          :level="3"
+          :title="
+            $t('Great! You account has been confirmed and it\'s active now.')
+          "
+          :description="
+            redirectTo &&
+            $t('You will be redirected in 5 seconds to finish the checkout...')
+          "
+        />
+        <SwErrorsList :list="errors" />
+      </div>
+    </SfLoader>
+    <SwButton
+      @click="$router.push($routing.getUrl('/'))"
+      class="sw-registration-confirm__home-btn"
+    >
+      <SfIcon icon="chevron_left" color="white" size="20px" />
+      {{ $t("Return to homepage") }}
+    </SwButton>
+  </div>
+</template>
+<script>
+import { ref, onMounted, watch, onDeactivated } from "@vue/composition-api"
+import { confirmAccountRegistration } from "@shopware-pwa/shopware-6-client"
+import {
+  getApplicationContext,
+  useBreadcrumbs,
+  useSessionContext,
+  useUser,
+} from "@shopware-pwa/composables"
+import { SfLoader, SfHeading, SfNotification, SfIcon } from "@storefront-ui/vue"
+
+export default {
+  name: "ResetPassword",
+  components: {
+    SfLoader,
+    SfHeading,
+    SfNotification,
+    SfIcon,
+    SwButton: () => import("@/components/atoms/SwButton.vue"),
+    SwErrorsList: () => import("@/components/SwErrorsList.vue"),
+  },
+  setup(props, { root }) {
+    const { setBreadcrumbs } = useBreadcrumbs()
+    const { refreshUser } = useUser()
+
+    const { apiInstance, i18n, router, routing } = getApplicationContext({
+      contextName: "AccountRegistrationConfirm",
+    })
+    setBreadcrumbs([
+      {
+        name: i18n.t("Email confirmation"),
+        path: "/registration/confirm",
+      },
+    ])
+    const {
+      context: {
+        query: { hash, em, redirectTo },
+      },
+    } = root
+
+    const success = ref(false)
+    const errors = ref([])
+    const loading = ref(true)
+    const redirectTimer = ref()
+    const redirectUser = (redirectTo) =>
+      setTimeout(() => {
+        router.push(redirectTo)
```

Comment:
is the domain correct here?

Reply:
we pass here only a path, which is being used in `router.push` once user is confirmed. It's definitely case to be checked 👍🏻

Reply:
I checked it and router pushes the previously configured path (prepended with current domain' path) as it's passed during registration process. so it's working great!